app/components/view-goal/progress-bar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const ProgressBarComponent = ({ rating, time }) => {
```

Comment:
I think in the context of a progress bar, `progress` would be a better name than `rating`.